pages/boosters/cachePage.ts

```diff
@@ -0,0 +1,23 @@
+import {browser, by, element, ElementFinder, protractor} from 'protractor';
+
+import {AbstractGreetingServicePage} from './abstractGreetingServicePage';
+
+export class CachePage extends AbstractGreetingServicePage {
+  private clearCache = element(by.id('clear'));
+  private cacheState = element(by.id('cache-state'));
+
+  constructor() {
+    super(browser.params.url.boosters.cache);
+  }
+
+  async clickClear(): Promise<void> {
+    return this.clearCache.click();
+  }
+
+  getCacheStateElement(cached = false): ElementFinder {
+    if (cached) {
+      return this.cacheState.element(by.className('label-info'));
+    }
+    return this.cacheState.element(by.className('label-warning'));
```

Comment:
The selectors here look pretty fragile. Isn't there a better way?

Reply:
Ah nevermind, I see now that this is under the `cacheState` element. All good :-)